python/hsfs/engine/python.py

```diff
@@ -206,7 +218,6 @@ def _sql_offline(
         hive_config: Optional[Dict[str, Any]] = None,
         arrow_flight_config: Optional[Dict[str, Any]] = None,
     ) -> Union[pd.DataFrame, pl.DataFrame]:
```

Comment:
What about these annotations? Do they work if polars is not installed?

Reply:
Yes there are just type hints are they would not cause any run time effects. I have tested it with the unit test and it was working. Still need to run some workflow tests and see if everything is fine.

https://stackoverflow.com/questions/41692473/does-python-type-hint-annotations-cause-some-run-time-effects